content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Falco Plugins Early Access
+date: 2021-10-12
+author: Mark Stemm
+---
```

Comment:
Probably we want to specify a predictable `slug` here

Reply:
Ok, added.